.platform.app.yaml

```
@@ -0,0 +1,99 @@
+name: sylius
+
+type: php:8.1
+
+relationships:
+    database: "db:mysql"
+
+disk: 2048
+
+mounts:
+    "/var/cache": "shared:files/cache"
+    "/var/log": "shared:files/log"
+    "/var/sessions": "shared:files/sessions"
+    "/public/uploads": "shared:files/uploads"
+    "/public/media": "shared:files/media"
+
+web:
+    locations:
+        "/":
+            root: "public"
+            passthru: "/index.php"
+            allow: true
+            expires: -1
+            scripts: true
+        '/assets/shop':
+            expires: 2w
+            passthru: true
+            allow: false
+            rules:
+                '\.(css|js|jpe?g|png|gif|svgz?|ico|bmp|tiff?|wbmp|ico|jng|bmp|html|pdf|otf|woff2|woff|eot|ttf|jar|swf|ogx|avi|wmv|asf|asx|mng|flv|webm|mov|ogv|mpe|mpe?g|mp4|3gpp|weba|ra|m4a|mp3|mp2|mpe?ga|midi?)$':
+                    allow: true
+        '/media/image':
+            expires: 2w
+            passthru: true
+            allow: false
+            rules:
+                '\.(jpe?g|png|gif|svgz?)$':
+                    allow: true
+        '/media/cache/resolve':
+            passthru: "/index.php"
+            expires: -1
+            allow: true
+            scripts: true
+        '/media/cache':
+            expires: 2w
+            passthru: true
+            allow: false
+            rules:
+                '\.(jpe?g|png|gif|svgz?)$':
+                    allow: true
+
+variables:
+    env:
+        APP_ENV: 'prod'
+        APP_DEBUG: 0
+        N_PREFIX: /app/.global
+
+build:
+    flavor: none
+
+dependencies:
+    nodejs:
+        n: "*"
+        yarn: "*"
+        gulp-cli: "*"
+    php:
+        composer/composer: '^2'
+
+crons:
+    cancel-unpaid-orders:
+        spec: "0 2 * * *"
+        cmd: "php bin/console sylius:cancel-unpaid-orders"
+    generate-promotion-coupons:
+        spec: "0 2 * * *"
+        cmd: "php bin/console sylius:promotion:generate-coupons"
+    php bin/console sylius:remove-expired-carts:
+        spec: "0 2 * * *"
+        cmd: "php bin/console sylius:cancel-unpaid-orders"
```

Not sure, that cron commands get `APP_ENV=prod`, better be safe, than sorry :smiley: 

```suggestion
        cmd: "php bin/console --env=prod sylius:cancel-unpaid-orders"
```